W-15523876 Update pricing-metrics.adoc (#241)
mulesoft/docs-general: modules/ROOT/pages/pricing-metrics.adoc

|Data Graph	
|_DataGraph orchestration_: An API request made by Anypoint DataGraph to the source APIs to get data for the GraphQL API request made to Anypoint DataGraph. 
|Orchestrations are not currently aggregated on usage reports.

|IDP 

> |Intelligent Document Processing (IDP) Document Pages: A single page processed by IDP.

Author: ```suggestion
|_Intelligent Document Processing (IDP) Document Pages_: A single page processed by IDP.
```

> |IDP document actions may process documents that have more than one page with each page counting separately. Executing document actions from RPA also counts Document Pages and additionally consumes the corresponding RPA Bot Minutes accounting for the time the RPA process runs. 

Reviewer: ```suggestion
|IDP document actions might process documents that have more than one page, with each page counting separately. When RPS executes document actions, it counts Document Pages and consumes the corresponding RPA Bot Minutes so that it also counts the time the RPA process runs. 
```

Reviewer: Is the capitalization correct for Document Pages? I know we typically don't capitalize the general use of a term, but this might be an exception.

The wording on this one is a bit hard to follow. I've made some suggestions, but of course, I leave it to you to determine what's best.

> |IDP document actions may process documents that have more than one page with each page counting separately. Executing document actions from RPA also counts Document Pages and additionally consumes the corresponding RPA Bot Minutes accounting for the time the RPA process runs. 

Author: ```suggestion
|IDP document actions might process documents that have more than one page, with each page counting separately. When RPA executes document actions, it also counts towards document pages and additionally consumes the corresponding RPA Bot Minutes, accounting for the time the RPA process runs. 
```

Reviewer: Thanks, Cris. Definitely easier to follow now. 
I still think you need commas before the clauses at the end of a couple of the sentences, because otherwise the sentence can be read incorrectly. Unfortunately, grammar checks aren't reliable for comma usage. For example, you need one between page and with in the first sentence and one between Minutes and accounting in the last sentence. I think you can safely follow this rule: Use commas wherever necessary to prevent possible confusion or misreading.
If you don't have time today to address this, I suggest asking the editorial team post-GA.
